src/SamplesApp/UITests.Shared/Windows_Devices/LampTests.xaml.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Windows.Input;
+using Uno.UI.Samples.Controls;
+using Uno.UI.Samples.UITests.Helpers;
+using Windows.Devices.Lights;
+using Windows.UI.Core;
+using Windows.UI.Xaml.Controls;
+
+namespace UITests.Shared.Windows_Devices
+{
+	[SampleControlInfo(
+		"Windows.Devices",
+		"Lamp",
```

Comment:
By convention we use the class name here.
(by the way it won't be required anymore once my PR #1726 is merged!)

Reply:
In the `Windows.Devices` category, all the samples have their name the same as their class name. In fact, I have done it this way for the majority of my PRs so far :-D .

Reply:
I mean `Lamp` vs `LampTests`.

Reply:
I meant that I have done it in similar fashion on the other sensors, for example here https://github.com/unoplatform/uno/blob/0430120103d6552443e2db7c5e0aaebcce059b2e/src/SamplesApp/UITests.Shared/Windows_Devices/BarometerTests.xaml.cs#L23 and here https://github.com/unoplatform/uno/blob/0430120103d6552443e2db7c5e0aaebcce059b2e/src/SamplesApp/UITests.Shared/Windows_Devices/GeolocatorTests.xaml.cs#L16